markdown/about.md

```diff
@@ -30,13 +30,15 @@ the nf-core alumni hall of fame:
 - [@KevinMenden](https://github.com/KevinMenden): Kevin Menden
 - [@olgabot](https://github.com/olgabot): Olga Botvinnik
 - [@sven1103](https://github.com/sven1103): Sven F.
+- [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
```

Comment:
Technically not core team. Can you perhaps just adjust the intro text to be less specific?

```markdown
The following people have earned their place in the nf-core alumni hall of fame:
```